content/en/blog/news/2022-04-22-k8s-122-tekton.md

```diff
@@ -57,3 +57,23 @@ In the current version of Tekton used by Jenkins X, the resource `requests` are
 To prepare for upgrade, you should create a PR applying the changes described above. We done some tests on applying the changes to the current version of Jenkins X/Tekton, and it seemed to work. But - that does not mean it will work on your machines or even pipelines! 
 
 So, please prepare for upgrade now! We want people to have a safe upgrade experience. If you have questions, you can find us on [Slack](https://kubernetes.slack.com/messages/C9MBGQJRH).
+
+#### Cannot resolve DNS hostname outside of cluster
+With the recent upgrade of base image of ghcr.io/jenkins-x/jx-boot:3.2.338 to alpine linux v3.15 in some of the local providers like minikube, the pod's DNS nameserver is not able to resolve DNS hostname outside k8s cluster like github.com. 
+
+To resolve this issue you need to add [custom dnspolicy](https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy) on your local cluster repository in below files with this change
+```yaml
```

Comment:
Is it possible to change the dns settings in the minikube cluster itself?

Reply:
yes it is possible and even I have raised a corresponding [PR](https://github.com/jx3-gitops-repositories/jx3-minikube/pull/7) . But it will be specific to minikube and as you pointed out before, these changes cannot be merged to the jenkins-x versionStream repo.

Reply:
I meant if you can change it when creating the minikube cluster (before running jx admin operator and running the boot job).

Reply:
With further testing, I found some more bugs and above solution is not enough. I'll work upon it in my free time. As of now please do not merge this PR.